tsl/src/continuous_aggs/materialize.c

```diff
@@ -307,6 +312,11 @@ spi_insert_materializations(Hypertable *mat_ht, SchemaAndName partial_view,
 
 	if (res < 0)
 		elog(ERROR, "could not materialize values into the materialization table");
+	else
+		elog(LOG,
+			 "inserted %lu row%s into materialization table",
+			 (unsigned long) SPI_processed,
+			 SPI_processed == 1 ? "" : "s");
```

Comment:
It could be helpful to print the table name here as well.

Reply:
Added table name